Carbon.IIS/Functions/Install-CIisVirtualDirectory.ps1

```diff
@@ -29,9 +29,10 @@ function Install-CIisVirtualDirectory
         [Parameter(Mandatory)]
         [String] $SiteName,
 
-        [Parameter(Mandatory)]
         # The virtual path of the virtual directory to install, i.e. the path in the URL to this directory. If creating
         # under an applicaton, this should be the path in the URL *after* the path in the URL to the application.
+        [Parameter(Mandatory)]
+        [Alias('Name')]

```

Comment:
Can have, eg. `-Name 'a/b'`? That reads a little off if so.

Reply:
Yes, which is why this is called `VirtualPath` now instead of `Name`. `Name` is deprecated and will be removed in v2.